src/test/java/org/fundacionjala/trello/pages/board/BoardForm.java

```diff
@@ -54,23 +57,30 @@ public boolean isDisplayed() {
         return data;
     }
 
+    @Override
+    public BoardPage submit() {
+        action.click(createButton);
+        return new BoardPage(driver);
+    }
+
     public BoardForm setTitle(final String title) {
         inputTitle.sendKeys(title);
         return this;
     }
 
     public BoardForm setTeam(final String team) {
         action.click(dropdownTeam);
-        String locator = String.format(XPATH_TEAM_OPTION, team);
-        WebElement teamSelected = driver.findElement(By.xpath(locator));
-        action.click(teamSelected);
-        wait.until(ExpectedConditions.invisibilityOf(teamSelected));
+        selectTeamOption(team);
         return this;
     }
 
-    @Override
-    public BoardPage submit() {
-        action.click(createButton);
-        return new BoardPage(driver);
+    private void selectTeamOption(final String team) {
+        String locator;
+        if (NO_TEAM.equals(team)) {
```

Comment:
this will mostly not match with the ones you send in the feature file, and will almost never execute